uiowa/uiowa
# oembed providers

Resolves https://github.com/uiowa/uiowa/issues/2356
Resolves https://github.com/uiowa/uiowa/issues/2667
Resolves https://github.com/uiowa/uiowa/issues/2720

Cleaning this up now as Youtube fixed the original issue. Drupal 9.2 has a more robust fallback to prevent future issues with youtube and other providers related to https://github.com/uiowa/uiowa/issues/2667, https://github.com/uiowa/uiowa/issues/2720, so I would advise moving to 9.2 sooner than later to prevent future disruption upstream.

# Stories

As a site editor I am still able to view existing remote video embeds and add new Kalutra Mediaspace public videos from mediaspace.uiowa.edu using the oembed share url.

As a webmaster I cannot access oembed providers configuration.

As a developer, my site syncs correctly and it is no longer dependent on a core patch/hooks.

As a site editor, I can throw any supported youtube link (public) at the remote video functionality and have it work.

# To Test

- sync a site with existing remote video embeds (e.g. sandbox.uiowa.edu)
- note that oembed providers module is enabled and config is imported.
- webmasters can't access the oembed providers configuration
- existing youtube and kaltura videos are still rendering
- you can add a youtube and kaltura video (e.g. https://mediaspace.uiowa.edu/id/1_hhz13skb)
- try various youtube urls schemes: https://www.youtube.com, https://youtube.com, https://youtu.be - you should not get `Could not retrieve the oEmbed resource`
